sdks/python/apache_beam/runners/common.py

```diff
@@ -661,8 +661,8 @@ def __init__(self, placeholder):
 
   def invoke_process(self,
                      windowed_value,  # type: WindowedValue
-                     restriction_tracker=None, # type: Optional[RestrictionTracker]
-                     watermark_estimator=None, # type: Optional[WatermarkEstimator]
+                     restriction=None,
```

Comment:
New type is just 'object' ?

Reply:
Yeah, since the user can choose anything they want for the restriction and watermark estimator state type.